invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.overrides

```diff
@@ -188,3 +188,7 @@
     padding-right: 0 !important;
   }
 }
+
+.ui.left.aligned.middle.grid.current-user {
+    margin-left: 12.5px;
+}
```

Comment:
what about using already implemented classes `ml-10` in order to avoid creating the custom CSS?